src/OpenAIGym.jl

```diff
@@ -1,6 +1,3 @@
-
- __precompile__()
-
 module OpenAIGym
 
 using PyCall
```

Comment:
this line wasn't added in this PR, but we need to add PyCall to Project.toml

Reply:
I added PyCall here
https://github.com/JuliaML/OpenAIGym.jl/pull/10/commits/3058d7e4af3761c8fb3c0728b89e5f65882fe07d#diff-910a7b3ca0075d545dcec45cb7335ca9R7